pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
     "Topic :: Other/Nonlisted Topic",
     "Topic :: Scientific/Engineering",
 ]
-requires-python = '>="3.8"'
+requires-python = '>=3.8'
```

Comment:
This fixes some weird upstream bug for parsing pyproject.

Reply:
I just made this change in main!